gke-deploy/core/resource/testing/deployment-3.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
```

Comment:
Are these YAMLs included in this PR on purpose?

Reply:
These YAMLs are used in unit tests